Formula/bitcoin.rb

```diff
@@ -1,9 +1,9 @@
 class Bitcoin < Formula
   desc "Decentralized, peer to peer payment network"
   homepage "https://bitcoin.org/"
-  url "https://github.com/bitcoin/bitcoin/archive/v0.15.1.tar.gz"
-  sha256 "e429d4f257f2b5b6d0caaf36ed1a5f5203e5918c57837efac00b0c322a1fef79"
-  head "https://github.com/bitcoin/bitcoin.git"
+  url "https://bitcoin.org/bin/bitcoin-core-0.15.1/bitcoin-0.15.1.tar.gz"
+  sha256 "34de2dbe058c1f8b6464494468ebe2ff0422614203d292da1c6458d6f87342b4"
+  revision 1
```

Comment:
Curious: what's the revision bump needed for?

Reply:
We generally revision bump if the tarball changed as a matter of course.

Reply:
Given the change is useless I'll just merge without the revision bump.